locations/spiders/neuroth.py

```
@@ -1,37 +1,19 @@
 
-from locations.hours import DAYS, OpeningHours
 from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
 
 
-class NeurothSpider(Spider):
+class NeurothSpider(SitemapSpider, StructuredDataSpider):
     name = "neuroth"
     item_attributes = {"brand": "Neuroth", "brand_wikidata": "Q15836645"}
+    sitemap_urls = ["https://www.neuroth.com/robots.txt"]
+    sitemap_rules = [(r".+/filialen|lokacije|poslovnice|/[^/]+/$", "parse_sd")]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)